ynput/OpenPype
# General: Use colorspace data when creating thumbnail

## Changelog Description
Thumbnails with applied colormanagement.

## Additional info
Originally developed by @tokejepsen at https://github.com/ynput/OpenPype/pull/5256
- added settings for default target values at oiio section default 

## TODO:
- [ ] ayon server settings

## Testing notes:
1. Create a render or review.
2. Publish and verify thumbnail is correct color.

- [x] Maya (farm render / review local)
- [x] Houdini (farm only)
- [x] Blender (farm only)
- [x] 3dsmax (farm only)
- [ ] TrayPublisher
- [ ] Substancepainter
- [ ] Nuke (local, farm)

## Explicit Thumbnai workflow > development stages
[original PR](https://github.com/ynput/OpenPype/pull/5936)
This PR is just a placeholder and will be broken into smaller chunks:
- Colormanged Thumbnails - https://github.com/ynput/OpenPype/pull/5938	
- Ftrack integrate instance components refactory of the plugin for support of multiple thumbnail representations. https://github.com/ynput/OpenPype/pull/5939
- Nuke Explicit Thumbnail workfow https://github.com/ynput/OpenPype/pull/5941:
   - adding `need-thumbnail` tag and cross plugin flow
   - Nuke: removing extract_thumbnail 
   - improving flow of `publish_on_farm` tags
- adding new settings and implementation of new functions into extract thumbnail https://github.com/ynput/OpenPype/pull/5944:
	- integrating thubnails switch 	
	- size constrain 
	- make thumbnail from middle
